src/common/types/enums/ecdsa.ts

```diff
@@ -0,0 +1,6 @@
+enum ECDSA {
```

Comment:
We're about to get rid of this folder, you should put this file in `src/common`.

Reply:
Sure.